.github/workflows/ci.yml

```diff
@@ -24,7 +24,9 @@ jobs:
       - name: Setup node env 📦
         uses: actions/setup-node@v3
         with:
-          node-version: 10
+          node-version: 17
```

Comment:
does this build at node 17? there are no maplibre-native binaries for that version so I would assume no.

Reply:
No, this workflow doesn’t test the code but rather ensure code style is followed along with code style checks for Dockerfile

Reply:
The CT workflow runs the tests in Actions